draft-ietf-quic-transport.md

```diff
@@ -7475,7 +7479,7 @@ Substantial editorial reorganization; no technical changes.
 - You don't always need the draining period (#871)
 - Stateless reset clarified as version-specific (#930, #986)
 - initial_max_stream_id_x transport parameters are optional (#970, #971)
-- Ack Delay assumes a default value during the handshake (#1007, #1009)
+- ACK delay assumes a default value during the handshake (#1007, #1009)
```

Comment:
```suggestion
- ACK Delay assumes a default value during the handshake (#1007, #1009)
```

Reply:
I didn't make changes to the changelogs, since they will be removed anyway. Should I have?

Reply:
This was a change,  I'm reverting it